addons/website/static/src/scss/website.scss

```diff
@@ -1082,3 +1082,7 @@ table.table_desc tr td {
 .modal-footer > .float-left {
     margin-right: auto;
 }
+
+.o_perk span.fa {
```

Comment:
maybe adding class `d-block` directly could be better?

Reply:
As a fix, it is probably better to keep the solution in scss instead of xml. However it should be in website_hr_recruitment, not website. I don't think o_perk exists in website ?

Reply:
There is no scss file in website_hr_recruitment. This is why I putted that there but I can create a new scss file if you prefer?

Reply:
@JKE-be better to add a new asset template and file in hr_recruitment I guess?

@jvm-odoo Was it better in < 12.0 ? How was it beautiful without this ? If it was already not perfect before, the FP request can be handled in master, not stable versions I guess (cc @JKE-be this is for the "default_website_description" view though, do you think we should improve in stable?)